packages/frontend/src/pages/scaling-activity/view/ActivityView.tsx

```diff
@@ -109,15 +122,24 @@ export function ActivityView({ items }: ActivityViewProps) {
   ]
 
   const rows: RowConfig<ActivityViewEntry> = {
-    getProps: (entry) =>
-      entry.name === 'Ethereum'
-        ? {
-            className: cx(
-              'bg-blue-400 hover:bg-blue-400 border-b-blue-600',
-              'dark:bg-blue-900 dark:border-b-blue-500 dark:hover:bg-blue-900',
-            ),
-          }
-        : {},
+    getProps: (entry) => {
+      let className = ''
+
+      if (entry.name === 'Ethereum') {
```

Comment:
There's no way that Ethereum is unverified. I think I'd like explicit returns in each of cases, there would be no need to create `className` variable then